app/views/admin/accounting/transactions/_form.slim

```diff
@@ -75,3 +74,13 @@
 
   .view-element = f.input :qb_record_id
     = @transaction.qb_id
+
+javascript:
+  $(function() {
+    $('#accounting_transaction_accounting_customer_id').select2({
```

Comment:
Using `f.association` in Simple Form requires a different way to initialize the Select2 autocomplete selects.